piano-hero/mobile/src/main/java/com/novoda/pianohero/PianoC4ToB5Activity.java

```diff
@@ -1,64 +1,34 @@
 package com.novoda.pianohero;
 
-import android.content.Context;
 import android.os.Bundle;
 import android.support.annotation.Nullable;
 import android.support.v7.app.AppCompatActivity;
-import android.view.KeyEvent;
 import android.view.Menu;
 import android.view.MenuItem;
-import android.view.inputmethod.InputMethodManager;
 
 import java.util.HashSet;
 import java.util.Set;
 
 public class PianoC4ToB5Activity extends AppCompatActivity {
 
-    private Brain brain;
-    private SimpleNotesOutputView outputView;
     private PianoC4ToB5View inputView;
+    private GamePresenter presenter;
 
     @Override
     protected void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_piano_c4_to_b5);
 
-        outputView = (SimpleNotesOutputView) findViewById(R.id.simple_notes_output_view);
         inputView = (PianoC4ToB5View) findViewById(R.id.piano_view);
-        brain = new Brain(onSequenceUpdatedCallback);
-
-        startNewGame();
-    }
-
-    private final OnSequenceUpdatedCallback onSequenceUpdatedCallback = new OnSequenceUpdatedCallback() {
-        @Override
-        public void onNext(Sequence sequence) {
-            outputView.display(sequence);
-        }
-    };
-
-    private void startNewGame() {
-        Sequence sequence = new SongSequenceFactory().maryHadALittleLamb();
-        brain.start(sequence);
+        GameMvp.View outputView = (SimpleNotesOutputView) findViewById(R.id.simple_notes_output_view);
```

Comment:
:x: this is the wrong view for this activity - it's supposed to be the PianoC4ToB5View